stock_available_to_promise_release/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    stock_reservation_horizon = fields.Integer(
+        default=15,
```

Comment:
Maybe extract this to a `_default_stock_reservation_horizon` method to allow easier override. Also I feel like the default value should be 0 instead of a defined number...

Reply:
there's no need for a method: you can override it by overriding the field.
ok for `0` by default.